### ISSUE TITLE
Bug – Reserved word with browserify

### ISSUE THREAD

**Issue by Buzut** [adamhalasz/uniqid#18]
This bug didn't exists in version 4.1.1. Now requiring the module with browserify (16.2.2) returns:

```
SyntaxError: unknown: interface is a reserved word in strict mode (475:14)
  473 |     var mac = '', interfaces = require('os').networkInterfaces();
  474 |     for(interface_key in interfaces){
> 475 |         const interface = interfaces[interface_key], length = interface.length;
      |               ^
  476 |         for(var i = 0; i < length; i++){
  477 |             if(interface[i].mac && interface[i].mac != '00:00:00:00:00:00'){
  478 |                 mac = interface[i].mac; break;
```

**Comment by adamhalasz** [comment]
Hi @Buzut thanks for the bug report. It should be fixed in the latest version `v5.0.3`.

Thanks,
Adam

**Comment by Buzut** [comment]
Thank you 👍
